vast/build.sh

```diff
@@ -32,7 +32,9 @@ fi
 
 initialise "${REVISION}" "${OUTPUT}" "${LAST_REVISION}"
 
-STAGING_DIR=/opt/compiler-explorer/vast-${VERSION}
+export STAGING_DIR=/opt/compiler-explorer/vast-${VERSION}
+export VCPKG_ROOT=/vcpkg
+export CMAKE_PREFIX_PATH=/usr/lib/llvm-18/
```

Comment:
No need for this, it is set from cmake preset `compiler-explorer`.

This dockerfile should be llvm version agnostic, so we can bump it up from within vast repository, i.e. without need of PR to compiler-explorer.

Reply:
The gap port gets its CMAKE_PREFIX_PATH from the environment, not from CMake cache values